modules/ROOT/pages/message-flows.adoc

```diff
@@ -1,9 +1,9 @@
-= Partner Manager Message Flows 
+= Partner Manager Message Flows
```

Comment:
What about them?
Can we make this title more precise?

Reply:
"Message Processing Using Inbound and Outbound Message Flows"?

Reply:
This is an about topic and this matches the UI, so for now, I think it should stay. I will ask the PM.